release-notes/next.yml

```
@@ -0,0 +1,70 @@
+- release: "35.20220313.1.0"
+  issues:
+    - id: "1048"
```

thoughts about using `url:` here instead. Since theoretically changes that we want to mention might come from somewhere else than just the issue tracker. 

```suggestion
    - url: "https://github.com/coreos/fedora-coreos-tracker/issues/1048"
```